tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/populate_route_ids_test.py

```diff
@@ -98,3 +98,30 @@ def test_integration_get_route_mapping_queries(
         self.assertEqual(queries[0][cdr_consts.DESTINATION_TABLE],
                          common.DRUG_EXPOSURE)
         self.assertEqual(queries[0][cdr_consts.DISPOSITION], 'WRITE_TRUNCATE')
+
+    def _test_integration_create_drug_route_mappings_table(self):
```

Comment:
Glad to see this moved! Also, I see this is defined above already, if you remove the underscore, so this can be removed.